articles/machine-learning/how-to-machine-learning-interpretability-aml.md

```diff
@@ -349,7 +349,7 @@ To load the visualization dashboard, use the following code.
 ```python
 from interpret_community.widget import ExplanationDashboard
 
-ExplanationDashboard(global_explanation, model, x_test)
+ExplanationDashboard(global_explanation, model)
```

Comment:
this used to be the third parameter, but now it is parameterized, you need to pass:
```
dataset=x_test
```
or
```
datasetX=x_test
```
otherwise, some visualizations (data exploration) will not be available in the widget

Reply:
Okay, I found what was confusing. First of all, we're introducing different explainers and local/remote executions and not providing end-to-end Notebooks, which makes it a little bit difficult to see how things are connected together. Second, the issue above is related with how `global_explanation` is created, and if you scroll up you find this:
```
# you can use the training data or the test data here
global_explanation = explainer.explain_global(x_train)  
```
But it turns out, you need to pass test data there if you want to do:
```
ExplanationDashboard(global_explanation_tab, model, dataset=x_test)
```

I confirm it works well when I follow your suggestion AND have calculated the explanation with test data. So I suggest we update the latter too.